releasenotes/notes/parallel-check-public-7faed5f3e20e1d03.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features_misc:
+  - |
+    A new function, :func:`qiskit.utils.should_run_in_parallel`, is available to determine whether
+    :func:`.parallel_map` will choose to run in parallel or degrade to running in serial.  This
```

Comment:
The word `degrade` seems a bit strange to me here. Maybe `fall back` would be more appropriate? I see that this word is also used in the docstring for `should_run_in_parallel` and would need to be updated as well.

Reply:
Completely changed this release note during the force push.